### ISSUE TITLE
Bug - Office.js Word Add-in Ribbon works only with Developer Window [Mac]

### ISSUE THREAD

**Issue by danlg** [OfficeDev/office-js#2433]
> Message from office-js bot:
We’re closing this issue because it has been inactive for a long time. We’re doing this to keep the issues list manageable and useful for everyone. If this issue is still relevant for you, please create a new issue.
> Thank you for your understanding and continued feedback.

Office Word Add in Ribbon work only with Developer Window
## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: Mac 16.57
* Host [Excel, Word, PowerPoint, etc.]: Word
* Office version number: 16.57
* Operating System: Mac BigSur
* Browser (if using Office on the web): N/A 
## Expected behavior

I would expect the add in to work when developer window is set to false (defaults write com.microsoft.Word OfficeWebAddinDeveloperExtras -bool false) and I sideload (yarn start:desktop)
https://docs.microsoft.com/en-us/office/dev/add-ins/testing/debug-office-add-ins-on-ipad-and-mac
For production, we cannot expect to have this flag set to true nor to have a developer Window !
 
## Steps to reproduce

1. Create a simple button with Ribbon Add in using yeoman
2. side load the addin yarn start:desktop
3. Click the button. Nothing happens

I have added a screen shot when it works with developer window
![bug_developer_window_mac](https://user-images.githubusercontent.com/12525045/154430900-e29be753-ec51-43a6-aea3-e6a59c318c38.jpg)

**Comment by danlg** [comment]
![bug_developer_window_mac](https://user-images.githubusercontent.com/12525045/154430900-e29be753-ec51-43a6-aea3-e6a59c318c38.jpg)

**Comment by danlg** [comment]
thanks @KelBowen this is blocking go to live. thanks for your help. Note : this happening on Word for Mac not Outlook ( I saw   macOutlookExtensibility assigned), although I may be at a later date, it's less urgent - interested to have this plugin in Outlook too. 

I am surprised to say it politely by the number of bugs I have found in Office @exextoc ... The critical issues I am experiencing and the issues I read from other LIVE users are scary....which raises the question fo the quality and stability of releases on https://appsforoffice.microsoft.com/lib/1/hosted/office.js.
